pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -73,9 +133,41 @@ SEED_NODES=("ade4d8bc8cbe014af6ebdf3cb7b1e9ad36f412c0@seeds.polkachu.com:23856",
 sed -i 's/seeds = ""/seeds = "'"${SEED_NODES[*]}"'"/' $HOME/.dydxprotocol/config/config.toml
 ```
 
-### Step 7: Create Service to Start Full Node
+The preceding command updates the `seeds` variable of `config.toml` with the list you provide.
+
+### Step 8: Download and extract a snapshot of the chain's history since genesis
+Using snapshots to restore or sync your full node's state saves time and effort. Using a snapshot avoids replaying all the blocks from genesis and does not require multiple binary versions for network upgrades. Instead, your node reads most of the chain's history directly from the snapshot.
+
+To download and extract the snapshot contents to the default dydxprotocol home directory, first **change directories into ./dydxprotocol**. To change directories, run the following command:
+
+```bash
+cd $HOME/.dydxprotocol
+```
 
-We will create a systemd service to start the full node.  This will allow the full node to start automatically on boot.
+Then, find a provider for your use case on the [Snapshot Service](https://docs.dydx.exchange/network/resources#snapshot-service) page.
+
+> For example, if you are connecting to `dydx-testnet-4`, you may use the provider [Bware Labs](https://bwarelabs.com/snapshots/dydx).
+
+Use the provider's instructions to download the snapshot into your current directory, `/.dydxprotocol`.
+
+> In most cases, you can run `wget snapshot-web-address`.
+
+From the current directory `$/HOME/.dydxprotocol`, run the following command, replacing the example value `your-snapshot-filename`:
+
+```bash
+lz4 -dc < your-snapshot-filename.tar.lz4 | tar xf -
+```
+The preceding command creates the `/data` folder in your current directory, `/.dydxprotocol`.
```

Comment:
I think this statement is inaccurate. I think the above command just uncompresses the snapshot file. 

you prob need to append `/data`? so something like:

`lz4 -dc < your-snapshot-filename.tar.lz4 | tar xf - -C "$HOME/.dydxprotocol/data"`

Reply:
I tested this pretty thoroughly -- could definitely have done something wonky myself, but I did notice this and double check because it seemed strange. will try it again and run it someone (probably Affan)